rockstar/dompurify-3.2.3.min.js

```diff
@@ -0,0 +1,3 @@
+/*! @license DOMPurify 3.2.3 | (c) Cure53 and other contributors | Released under the Apache license 2.0 and Mozilla Public License 2.0 | github.com/cure53/DOMPurify/blob/3.2.3/LICENSE */
```

Comment:
Sourced the file from Dompurify's repo https://github.com/cure53/DOMPurify/blob/ad0fe79863f24d9a6e6583a350e2efb791f77bc9/dist/purify.min.js#L1